The shippable failure can't be due to this PR:
```
--- FAIL: TestExampleObjectSchemas (0.54s)
examples_test.go:344: skipping : ../docs/user-guide/multi-pod.yaml/multi-pod
examples_test.go:339: ../examples/pod.yaml: pod does not have a test case defined
examples_test.go:344: skipping : ../examples/elasticsearch/apiserver-secret.yaml/apiserver-secret
FAIL
```
@mikedanese

--- a/docs/user-guide/configuring-containers.md
+++ b/docs/user-guide/configuring-containers.md
@@ -32,6 +32,7 @@ certainly want the docs that go with that version.</h1>
   - [Viewing pod status](#viewing-pod-status)
   - [Viewing pod output](#viewing-pod-output)
   - [Deleting pods](#deleting-pods)
+  - [What's next?](#what's-next?)
 
 <!-- END MUNGE: GENERATED_TOC -->
 
@@ -165,6 +166,10 @@ pods/hello-world
 Terminated pods aren’t currently automatically deleted, so that you can observe their final status, so be sure to clean up your dead pods. 
 
 On the other hand, containers and their logs are eventually deleted automatically in order to free up disk space on the nodes.
+
+## What's next?
+
+[Learn about deploying continuously running applications.](deploying-applications.md)
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->

--- a/docs/user-guide/connecting-applications.md
+++ b/docs/user-guide/connecting-applications.md
@@ -32,6 +32,7 @@ certainly want the docs that go with that version.</h1>
     - [Environment Variables](#environment-variables)
     - [DNS](#dns)
   - [Exposing the Service to the internet](#exposing-the-service-to-the-internet)
+  - [What's next?](#what's-next?)
 
 <!-- END MUNGE: GENERATED_TOC -->
 
@@ -236,6 +237,10 @@ $ kubectl get service nginxsvc -o json | grep \"ip\"
 "ip": "104.197.37.222"
 ```
 Now you have a load balancer that automatically does what you would’ve in the previous step. Note that you cannot directly curl your nodes on port 80, you need to go to the ip of the load balancer.
+
+## What's next?
+
+[Learn about more Kubernetes features that will help you run containers reliably in production.](production-pods.md)
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->

--- a/docs/user-guide/connecting-to-applications-port-forward.md
+++ b/docs/user-guide/connecting-to-applications-port-forward.md
@@ -21,8 +21,8 @@ certainly want the docs that go with that version.</h1>
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 #Connecting to applications: kubectl port-forward
-kubectl port-forward forwards connections to a local port to a port on a pod. Its man page is available [here](kubectl/kubectl_port-forward.md). Compared to [kubectl proxy](accessing-the-cluster.md#using-kubectl-proxy), `kubectl port-forward` is more generic as it can forward TCP traffic while `kubectl proxy` can only forward HTTP traffic. This guide demonstrates how to use `kubectl port-forward` to connect to a Redis database, which may be useful for database debugging.
 
+kubectl port-forward forwards connections to a local port to a port on a pod. Its man page is available [here](kubectl/kubectl_port-forward.md). Compared to [kubectl proxy](accessing-the-cluster.md#using-kubectl-proxy), `kubectl port-forward` is more generic as it can forward TCP traffic while `kubectl proxy` can only forward HTTP traffic. This guide demonstrates how to use `kubectl port-forward` to connect to a Redis database, which may be useful for database debugging.
 
 ## Creating a Redis master
 ```

--- a/docs/user-guide/deploying-applications.md
+++ b/docs/user-guide/deploying-applications.md
@@ -22,6 +22,17 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Kubernetes User Guide: Managing Applications: Deploying continuously running applications
 
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [Kubernetes User Guide: Managing Applications: Deploying continuously running applications](#kubernetes-user-guide:-managing-applications:-deploying-continuously-running-applications)
+  - [Launching a set of replicas using a configuration file](#launching-a-set-of-replicas-using-a-configuration-file)
+  - [Viewing replication controller status](#viewing-replication-controller-status)
+  - [Deleting replication controllers](#deleting-replication-controllers)
+  - [Labels](#labels)
+  - [What's next?](#what's-next?)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
 You previously read about how to quickly deploy a simple replicated application using [`kubectl run`](quick-start.md) and how to configure and launch single-run containers using pods (configuring-containers.md). Here, you’ll use the configuration-based approach to deploy a continuously running, replicated application.
 
 ## Launching a set of replicas using a configuration file
@@ -112,6 +123,10 @@ map[app:nginx]
 ```
 
 You could also specify the `selector` explicitly, such as if you wanted to specify labels in the pod template that you didn’t want to select on, but you should ensure that the selector will match the labels of the pods created from the pod template, and that it won’t match pods created by other replication controllers. The most straightforward way to ensure the latter is to create a unique label value for the replication controller, and to specify it in both the pod template’s labels and in the selector.
+
+## What's next?
+
+[Learn about exposing applications to users and clients, and connecting tiers of your application together.](connecting-applications.md)
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->

--- a/docs/user-guide/introspection-and-debugging.md
+++ b/docs/user-guide/introspection-and-debugging.md
@@ -32,6 +32,7 @@ your pods. But there are a number of ways to get even more information about you
   - [Using ```kubectl describe pod``` to fetch details about pods](#using-```kubectl-describe-pod```-to-fetch-details-about-pods)
   - [Example: debugging Pending Pods](#example:-debugging-pending-pods)
   - [Example: debugging a down/unreachable node](#example:-debugging-a-down/unreachable-node)
+  - [What's next?](#what's-next?)
 
 <!-- END MUNGE: GENERATED_TOC -->
 
@@ -316,6 +317,15 @@ status:
     osImage: Debian GNU/Linux 7 (wheezy)
     systemUUID: ABE5F6B4-D44B-108B-C46A-24CCE16C8B6E
 ```
+
+## What's next?
+
+Learn about additional debugging tools, including:
+* [Logging](logging.md)
+* [Monitoring](monitoring.md)
+* [Getting into containers via `exec`](getting-into-containers.md)
+* [Connecting to containers via proxies](connecting-to-applications-proxy.md)
+* [Connecting to containers via port forwarding](connecting-to-applications-port-forward.md)
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->

--- a/docs/user-guide/managing-deployments.md
+++ b/docs/user-guide/managing-deployments.md
@@ -36,6 +36,7 @@ You’ve deployed your application and exposed it via a service. Now what? Kuber
   - [Updating your application without a service outage](#updating-your-application-without-a-service-outage)
   - [In-place updates of resources](#in-place-updates-of-resources)
   - [Disruptive updates](#disruptive-updates)
+  - [What's next?](#what's-next?)
 
 <!-- END MUNGE: GENERATED_TOC -->
 
@@ -395,6 +396,10 @@ $ kubectl replace -f nginx-rc.yaml --force
 replicationcontrollers/my-nginx-v4
 replicationcontrollers/my-nginx-v4
 ```
+
+## What's next?
+
+[Learn about how to use `kubectl` for application introspection and debugging.](introspection-and-debugging.md)
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->

--- a/docs/user-guide/prereqs.md
+++ b/docs/user-guide/prereqs.md
@@ -26,6 +26,10 @@ To deploy and manage applications on Kubernetes, you’ll use the Kubernetes com
 
 In order for kubectl to find and access the Kubernetes cluster, it needs a [kubeconfig file](kubeconfig-file.md), which is created automatically when creating a cluster using kube-up.sh (see the [getting started guides](../../docs/getting-started-guides/) for more about creating clusters). If you need access to a cluster you didn’t create, see the [Sharing Cluster Access document](sharing-clusters.md).
 
+## What's next?
+
+[Learn how to launch and expose your application.](quick-start.md)
+
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
 [![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/user-guide/prereqs.md?pixel)]()

--- a/docs/user-guide/production-pods.md
+++ b/docs/user-guide/production-pods.md
@@ -22,6 +22,21 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Kubernetes User Guide: Managing Applications: Working with pods and containers in production
 
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [Kubernetes User Guide: Managing Applications: Working with pods and containers in production](#kubernetes-user-guide:-managing-applications:-working-with-pods-and-containers-in-production)
+  - [Persistent storage](#persistent-storage)
+  - [Distributing credentials](#distributing-credentials)
+  - [Authenticating with a private image registry](#authenticating-with-a-private-image-registry)
+  - [Helper containers](#helper-containers)
+  - [Resource management](#resource-management)
+  - [Liveness and readiness probes (aka health checks)](#liveness-and-readiness-probes-(aka-health-checks))
+  - [Lifecycle hooks and termination notice](#lifecycle-hooks-and-termination-notice)
+  - [Termination message](#termination-message)
+  - [What's next?](#what's-next?)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
 You’ve seen [how to configure and deploy pods and containers](configuring-containers.md), using some of the most common configuration parameters. This section dives into additional features that are especially useful for running applications in production.
 
 ## Persistent storage
@@ -335,6 +350,10 @@ Sleep expired
 $ kubectl get pods/pod-w-message -o template -t "{{range .status.containerStatuses}}{{.lastState.terminated.exitCode}}{{end}}"
 0
 ```
+
+## What's next?
+
+[Learn more about managing deployments.](managing-deployments.md)
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->

--- a/docs/user-guide/quick-start.md
+++ b/docs/user-guide/quick-start.md
@@ -22,6 +22,16 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Kubernetes User Guide: Managing Applications: Quick start
 
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [Kubernetes User Guide: Managing Applications: Quick start](#kubernetes-user-guide:-managing-applications:-quick-start)
+  - [Launching a simple application](#launching-a-simple-application)
+  - [Exposing your application to the Internet](#exposing-your-application-to-the-internet)
+  - [Killing the application](#killing-the-application)
+  - [What's next?](#what's-next?)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
 This guide will help you get oriented to Kubernetes and running your first containers on the cluster.
 
 ## Launching a simple application
@@ -73,6 +83,10 @@ replicationcontrollers/my-nginx
 $ kubectl delete svc my-nginx
 services/my-nginx
 ```
+
+## What's next?
+
+[Learn about how to configure common container parameters, such as commands and environment variables.](configuring-containers.md)
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->

